src/EditorFeatures/VisualBasicTest/Diagnostics/InsertMissingCast/InsertMissingCastTests.vb

```diff
@@ -149,8 +149,8 @@ End Module",
 "Option Strict On
 Module M1
     Function goo() As Integer
-        Return CInt(""10"") 
- End Function
+        Return CInt(""10"")
+    End Function
```

Comment:
:bug: Should not have indented `End Function` here (another case of previous bug).